admin/cluster_setup.py

```diff
@@ -11,23 +11,159 @@
 from pipes import quote as shell_quote
 
 from eliot import FileDestination, add_destination, write_failure
+from pyrsistent import pvector
+from txeffect import perform
 
-from twisted.internet.defer import inlineCallbacks
-from twisted.python.usage import UsageError
+from twisted.internet.defer import DeferredList, inlineCallbacks
 from twisted.python.filepath import FilePath
+from twisted.python.usage import UsageError
 
 from .acceptance import (
+    CLOUD_PROVIDERS,
     ClusterIdentity,
     CommonOptions,
+    LibcloudRunner as OldLibcloudRunner,
```

Comment:
Importing with an alias then subclassing using that alias is a slightly odd pattern. Why not just give a descriptive subclass name?